src/components/Navbar.tsx

```diff
@@ -106,6 +115,7 @@ function Navbar() {
   };
 
   return (
+    <div className="App" style={bodyStyle}>
```

Comment:
Why is this div added here?

Reply:
Actually for applying the CSS, to the whole webpage. Then I integrated the css code to  main.css. I removed this portion  now. 

I think now it is perfect.